package.json

```diff
@@ -35,8 +35,7 @@
   "files": [
     "LICENSE",
     "README.md",
-    "index.js",
-    "_locales/.gitkeep"
+    "index.js"
```

Comment:
We sure we don't need to include src/_locale folder? I think our installing works fine previously because we deploy as NPM package? Now it doesn't work is because we reference as repo?
@wildtron @wilbertverayin ??

Reply:
we'll use translation as default folder now